wandb/util.py

```diff
@@ -73,6 +72,7 @@
 #
 # TODO(spencerpearson): this should probably be replaced with pathlib.Path
 FilePathStr = NewType("FilePathStr", str)
+PathOrStr = Union[os.PathLike, FilePathStr]
```

Comment:
why do we need both?

Reply:
Why do we need both a FilePathStr and a PathOrStr? Or why are we using the FilePathStr instead of a generic str?

Reply:
the second one. i.e. why not just have: `PathOrStr = Union[os.PathLike, str]`